<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="G-0439" name="srm_Genesys_MarketingSync.SRM_Genesys_MarketingSync" tests="9" failures="1" timestamp="8 Apr 2016 08:30:09 GMT" time="120.269" errors="6">
  <testcase name="SearchStudentAfterPhoneModification_IWD" time="0.000" classname="srm_Genesys_MarketingSync.SRM_Genesys_MarketingSync">
    <error type="java.lang.Throwable" message="Method SRM_Genesys_MarketingSync.SearchStudentAfterPhoneModification_IWD(java.lang.reflect.Method)[pri:0, instance:srm_Genesys_MarketingSync.SRM_Genesys_MarketingSync@1ce4608] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method SRM_Genesys_MarketingSync.SearchStudentAfterPhoneModification_IWD(java.lang.reflect.Method)[pri:0, instance:srm_Genesys_MarketingSync.SRM_Genesys_MarketingSync@1ce4608] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1052)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1176)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1101)
	at org.testng.TestNG.run(TestNG.java:1009)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- SearchStudentAfterPhoneModification_IWD -->
  <testcase name="SearchStudentidSRM" time="0.000" classname="srm_Genesys_MarketingSync.SRM_Genesys_MarketingSync">
    <error type="java.lang.Throwable" message="Method SRM_Genesys_MarketingSync.SearchStudentidSRM(java.lang.reflect.Method)[pri:0, instance:srm_Genesys_MarketingSync.SRM_Genesys_MarketingSync@1ce4608] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method SRM_Genesys_MarketingSync.SearchStudentidSRM(java.lang.reflect.Method)[pri:0, instance:srm_Genesys_MarketingSync.SRM_Genesys_MarketingSync@1ce4608] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1052)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1176)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1101)
	at org.testng.TestNG.run(TestNG.java:1009)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- SearchStudentidSRM -->
  <testcase name="ChangeVendorNameSRMAsOriginal" time="0.000" classname="srm_Genesys_MarketingSync.SRM_Genesys_MarketingSync">
    <error type="java.lang.Throwable" message="Method SRM_Genesys_MarketingSync.ChangeVendorNameSRMAsOriginal(java.lang.reflect.Method)[pri:0, instance:srm_Genesys_MarketingSync.SRM_Genesys_MarketingSync@1ce4608] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method SRM_Genesys_MarketingSync.ChangeVendorNameSRMAsOriginal(java.lang.reflect.Method)[pri:0, instance:srm_Genesys_MarketingSync.SRM_Genesys_MarketingSync@1ce4608] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1052)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1176)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1101)
	at org.testng.TestNG.run(TestNG.java:1009)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- ChangeVendorNameSRMAsOriginal -->
  <testcase name="ChangeVendorNameSRM" time="0.000" classname="srm_Genesys_MarketingSync.SRM_Genesys_MarketingSync">
    <error type="java.lang.Throwable" message="Method SRM_Genesys_MarketingSync.ChangeVendorNameSRM(java.lang.reflect.Method)[pri:0, instance:srm_Genesys_MarketingSync.SRM_Genesys_MarketingSync@1ce4608] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method SRM_Genesys_MarketingSync.ChangeVendorNameSRM(java.lang.reflect.Method)[pri:0, instance:srm_Genesys_MarketingSync.SRM_Genesys_MarketingSync@1ce4608] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1052)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1176)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1101)
	at org.testng.TestNG.run(TestNG.java:1009)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- ChangeVendorNameSRM -->
  <testcase name="FetchDBmkChannelUpdate" time="14.807" classname="srm_Genesys_MarketingSync.SRM_Genesys_MarketingSync"/>
  <testcase name="SearchStudent_IWD" time="0.000" classname="srm_Genesys_MarketingSync.SRM_Genesys_MarketingSync">
    <error type="java.lang.Throwable" message="Method SRM_Genesys_MarketingSync.SearchStudent_IWD(java.lang.reflect.Method)[pri:0, instance:srm_Genesys_MarketingSync.SRM_Genesys_MarketingSync@1ce4608] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method SRM_Genesys_MarketingSync.SearchStudent_IWD(java.lang.reflect.Method)[pri:0, instance:srm_Genesys_MarketingSync.SRM_Genesys_MarketingSync@1ce4608] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1052)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1176)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1101)
	at org.testng.TestNG.run(TestNG.java:1009)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- SearchStudent_IWD -->
  <testcase name="CreateLeadImport" time="0.000" classname="srm_Genesys_MarketingSync.SRM_Genesys_MarketingSync">
    <error type="java.lang.Throwable" message="Method SRM_Genesys_MarketingSync.CreateLeadImport(java.lang.reflect.Method)[pri:0, instance:srm_Genesys_MarketingSync.SRM_Genesys_MarketingSync@1ce4608] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method SRM_Genesys_MarketingSync.CreateLeadImport(java.lang.reflect.Method)[pri:0, instance:srm_Genesys_MarketingSync.SRM_Genesys_MarketingSync@1ce4608] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1052)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1176)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1101)
	at org.testng.TestNG.run(TestNG.java:1009)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- CreateLeadImport -->
  <testcase name="SearchVendoridSRM" time="69.725" classname="srm_Genesys_MarketingSync.SRM_Genesys_MarketingSync">
    <failure type="java.lang.AssertionError" message="expected [Aggregator] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [Aggregator] but found [null]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at org.testng.Assert.assertEquals(Assert.java:188)
	at srm_Genesys_MarketingSync.SRM_Genesys_MarketingSync.SearchVendoridSRM(SRM_Genesys_MarketingSync.java:336)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:648)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:834)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1142)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1176)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1101)
	at org.testng.TestNG.run(TestNG.java:1009)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- SearchVendoridSRM -->
  <testcase name="Search_mkMarketingchannelid" time="35.737" classname="srm_Genesys_MarketingSync.SRM_Genesys_MarketingSync"/>
</testsuite> <!-- srm_Genesys_MarketingSync.SRM_Genesys_MarketingSync -->
